addons/mail/static/src/components/media_preview/media_preview.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.MediaPreview" owl="1">
+        <div class="o-MediaPreview position-relative d-flex justify-content-center">
+            <video class="o-MediaPreview-videoDisplay shadow rounded bg-dark" height="480" width="640" autoplay="" t-ref="video"/>
+
+            <p t-if="mediaPreview.doesBrowserSupportMediaDevices and !mediaPreview.isVideoEnabled" class="o-MediaPreview-mediaDevicesStatus position-absolute text-light">
+                Camera is off
+            </p>
+            <p t-if="!mediaPreview.doesBrowserSupportMediaDevices" class="o-MediaPreview-mediaDevicesStatus position-absolute text-light">
+                Your browser does not support videoconference
```

Comment:
Missing dot at the end, missing space after video?

Reply:
I didn't put a dot to stick to the specs! I'm going to change it if you think it's necessary.
The absence of space is intended, and it doesn't seem to be a mistake in English